README.md

```diff
@@ -151,6 +151,7 @@
 
 - [Gitly](https://github.com/vlang/gitly) - A light and fast SCM alternative to GitHub/GitLab written in V.
 - [Heroku Buildpack for V](https://github.com/zztkm/heroku-buildpack-v) - Deploy V apps on Heroku.
+- [Mantis](https://github.com/khalyomede/mantis) - a web framework written in V.
```

Comment:
```suggestion
- [Mantis](https://github.com/khalyomede/mantis) - A web framework written in V.
```
This will fix the CI failure.

Reply:
fixed with b443fc5a15494effdb0f92b669bedbfcd8c92193